Automattic/jetpack
# Move/service icons to js components package

#### Changes proposed in this Pull Request:
Moved SocialServiceIcon component from Jetpack Icons.js file to js-package/components. Updated it's ref in the Jetpack plugin directory



#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Make sure `pnpm build-extensions` is running fine in Jetpack plugin directory
* Add a new social connection in Marketing>Connections dashboard
* Create a new post and open the Jetpack sidebar from the top left to share this post. Ensure that the connected service icon is getting rendered properly

<img width="297" alt="Screenshot 2022-04-05 at 9 18 45 PM" src="https://user-images.githubusercontent.com/95693514/161794217-918ec2f2-8899-4854-9dac-31bb45d87fc3.png">
